fineract-provider/src/main/java/org/apache/fineract/portfolio/savings/api/SavingsProductsApiResourceSwagger.java

```diff
@@ -67,11 +67,46 @@ private PostSavingsCharges() {}
         public Integer interestCalculationType;
         @Schema(example = "365")
         public Integer interestCalculationDaysInYearType;
+        @Schema(example = "accountMappingForPayment")
```

Comment:
Seems unrelated change... should it be extracted into a separate PR?

Reply:
All the changes added in the Swagger classes were to have the base attributes to cover the POST request used in the ITC